depscan/cli.py

```diff
@@ -519,16 +519,17 @@ async def run_scan():
         project_type = q.get("type")
     if q.get("profile"):
         profile = q.get("profile")
-    if not url and params.get("url"):
-        url = params.get("url")
-    if not path and params.get("path"):
-        path = params.get("path")
-    if not multi_project and params.get("multiProject"):
-        multi_project = params.get("multiProject", "").lower() in ("true", "1")
-    if not project_type and params.get("type"):
-        project_type = params.get("type")
-    if not profile and params.get("profile"):
-        profile = params.get("profile")
+    if params is not None:
+        if not url and params.get("url"):
+            url = params.get("url")
+        if not path and params.get("path"):
+            path = params.get("path")
+        if not multi_project and params.get("multiProject"):
+            multi_project = params.get("multiProject", "").lower() in ("true", "1")
+        if not project_type and params.get("type"):
+            project_type = params.get("type")
+        if not profile and params.get("profile"):
+            profile = params.get("profile")
```

Comment:
As discussed, can we accept the bom file as a file upload?

Reply:
Ack.